Caddyfile

```diff
@@ -1,6 +1,8 @@
 0.0.0.0
 proxy / registration-provider:3000
-proxy /team teams-provider:3000
+proxy /teams teams-provider:3000 {
+    without /teams
```

Comment:
so no index?

Reply:
What do you mean by index? and why do we need it?

Reply:
So why teams and no team again?